index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="http://localhost:3400/style.css">
+    <title>FECDemo</title>
+  </head>
+  <body>
+    <div id="title-map"></div>
+    <div id="reviews"></div>
+    <div id="pictures"></div>
+    <div id="sidebar"></div>
+
+    <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
+    <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
+    <script src="http://localhost:3001/bundle.js"></script>
+    <script src="http://localhost:8001/bundle.js"></script>
+    <script src="http://localhost:3000/bundle.js"></script>
+    <script src="http://localhost:3400/bundle.js"></script>
+    <!-- see: https://reactjs.org/docs/react-without-jsx.html -->
+    <script>
+      ReactDOM.render(
+        React.createElement(TitleMap, {id: 101}, null),
+        document.getElementById('title-map')
+      );
+      ReactDOM.render(
+        React.createElement(Reviews, {id: 101}, null),
```

Comment:
I think I called mine ReviewApp, so you won't get mine with this element name, it'll show up undefined